pyanaconda/installation_tasks.py

```diff
@@ -19,7 +19,7 @@
 #
 from threading import RLock
 from pyanaconda.isignal import Signal
```

Comment:
I think that `isignal` can be also part of the core and that we can remove the `i`.

Reply:
ok